README.rst

```diff
@@ -550,6 +550,11 @@ Decorator example
 Changelog
 ---------
 
+[NEXT_RELEASE] - 2019-02-06
```

Comment:
Could you remove that date? 
simple-settings uses `bumpversion` to add the release date.

Reply:
sure, sorry i wasn't sure of your release process